src/components/Pages/PageFav.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import TemplateDefault from '../Templates/TemplateDefault';
+import MovieResult from '../Atoms/MovieResult';
+
+const Fav = () => {
+  const favMovieList = Object.values(localStorage)
+    .map((value) => {
+      try {
+        return JSON.parse(value);
+      } catch (e) {
+        return null;
+      }
+    })
+    .filter((movie) => {
+      if (movie && typeof movie === 'object' && movie.imdbID) {
+        return true;
+      }
+
+      return false;
+    });
+
+  if (localStorage.length === 0) {
```

Comment:
Problem here is if a plugin injects something the local storage will have an item (false positive). `favMovieList ` should be used instead of `localStoarge` for length check.